addons/l10n_in/views/report_invoice.xml

```diff
@@ -30,17 +30,19 @@
             </td>
         </xpath>
 
-        <xpath expr="//h2" position="replace">
-            <h2>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'posted'" t-field="o.journal_id.name"/>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'draft'">Draft <span t-field="o.journal_id.name"/></span>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'cancel'">Cancelled <span t-field="o.journal_id.name"/></span>
-                <span t-if="o.move_type == 'out_refund'">Credit Note</span>
-                <span t-if="o.move_type == 'in_refund'">Vendor Credit Note</span>
-                <span t-if="o.move_type == 'in_invoice'">Vendor Bill</span>
-                <span t-field="o.name"/>
-            </h2>
-        </xpath>
+        <t t-if="o.company_id.account_fiscal_country_id.code == 'IN'">
```

Comment:
I have never seen a t t-if over an xpath like that.  That works?

Reply:
We might add a t-if on the original h2 and add this one after...   I could not find another case in the code.  We are a bit mingling things here.  (xpath which are like static in the XML, while the t-if is dynamic)  @nle-odoo